frontend/app/components/checkin-dialog.js

```diff
@@ -2,8 +2,7 @@ import Ember from 'ember';
 import StepControl from 'flaredown/mixins/step-control';
 
 export default Ember.Component.extend(StepControl, {
-
-  classNameBindings: ['isSummary:checkin-summary:flaredown-white-box'],
+  tagName: '',
```

Comment:
Is there any reason to use an empty `tagName` and have `<div class="checkin-summary">` in the template, rather than simply having `classNames: 'checkin-summary'` here?
Also, why was the conditional logic based on `isSummary` removed?